presentation/src/main/java/com/kdjj/presentation/viewmodel/recipeeditor/RecipeEditorViewModel.kt

```diff
@@ -61,48 +63,125 @@ internal class RecipeEditorViewModel @Inject constructor(
                 .onSuccess { recipeTypes ->
                     _liveRecipeTypes.value = recipeTypes
-                    recipeId?.let {
-                        val recipeFlow =
-                            getLocalRecipeFlowUseCase(GetLocalRecipeFlowRequest(recipeId))
-                        val recipe = recipeFlow.first()
-                        val (metaModel, stepList) =
-                            recipe.toPresentation(recipeValidator, recipeTypes, recipeStepValidator)
-                        recipeMetaModel = metaModel
-                        _liveStepModelList.value = stepList
-                        isEditing = true
-
-                    } ?: run {
-                        recipeMetaModel = RecipeEditorItem.RecipeMetaModel.create(
-                            idGenerator, recipeValidator
-                        )
-                        _liveStepModelList.value = listOf(
-                            RecipeEditorItem.RecipeStepModel.create(
-                                idGenerator, recipeStepValidator
-                            )
-                        )
+                    fetchRecipeTempUseCase(FetchRecipeTempRequest(recipeId)).getOrNull()?.let {
+                        tempRecipe = it
+                        _eventRecipeEditor.value = Event(RecipeEditorEvent.TempDialog(recipeId))
+                    } ?: if (recipeId == NEW_ID) {
+                        createNewRecipe()
+                    } else {
+                        loadFromLocal(recipeId)
                     }
                 }
                 .onFailure {
                     _eventRecipeEditor.value = Event(RecipeEditorEvent.Error)
                 }
             _liveLoading.value = false
         }
+    }
+
+    fun showRecipeFromTemp() {
+        if (isInitialized) return
+        val (metaModel, stepList) =
+            tempRecipe.toPresentation(
+                recipeValidator,
+                _liveRecipeTypes.value ?: listOf(),
+                recipeStepValidator
+            )
+        recipeMetaModel = metaModel
+        _liveStepModelList.value = stepList
+        isInitialized = true
+    }
+
+    fun showRecipeFromLocal(recipeId: String) {
+        if (isInitialized) return
+        if (recipeId == NEW_ID) {
+            createNewRecipe()
+        } else {
+            viewModelScope.launch {
+                loadFromLocal(recipeId)
+            }
+        }
         isInitialized = true
     }
 
+    private fun createNewRecipe() {
+        recipeMetaModel = RecipeEditorItem.RecipeMetaModel.create(
+            idGenerator, recipeValidator
+        )
+        _liveStepModelList.value = listOf(
+            RecipeEditorItem.RecipeStepModel.create(
+                recipeStepValidator
+            )
+        )
+        viewModelScope.launch {
+            deleteRecipeTempUseCase(DeleteRecipeTempRequest(NEW_ID)).onSuccess {
+            }.onFailure {
+                it.printStackTrace()
+            }
```

Comment:
특별한 기능 추가가 아닌 확인용으로 작성하신 것 같은데 삭제 부탁드려요!
```suggestion
        viewModelScope.launch {
            deleteRecipeTempUseCase(DeleteRecipeTempRequest(NEW_ID))
```